server/services/Mailer.js

```diff
@@ -3,24 +3,22 @@ const helper = sendgrid.mail;
 const keys = require('../config/keys');
 
 class Mailer extends helper.Mail {
-  constructor({ subject, recipients }, content) {
+  constructor(survey, content) {
     super();
 
     this.sgApi = sendgrid(keys.sendGridKey);
     this.from_email = new helper.Email('no-reply@emaily.com');
-    this.subject = subject;
+    this.subject = survey.subject;
     this.body = new helper.Content('text/html', content);
-    this.recipients = this.formatAddresses(recipients);
+    this.recipients = this.formatAddresses(survey.recipients);
 
     this.addContent(this.body);
     this.addClickTracking();
     this.addRecipients();
   }
 
   formatAddresses(recipients) {
-    return recipients.map(({ email }) => {
-      return new helper.Email(email);
-    });
+    return recipients.map(recipient => new helper.Email(recipient.email));
```

Comment:
For simple arrow function callbacks, you can make it a single line and omit the curly brace and `return` keyword, which I find easier to read.